server/schedule/operator/step.go

```diff
@@ -672,14 +673,3 @@ func (cpl ChangePeerV2Leave) CheckInProgress(cluster opt.Cluster, region *core.R
 
 // Influence calculates the store difference that current step makes.
 func (cpl ChangePeerV2Leave) Influence(opInfluence OpInfluence, region *core.RegionInfo) {}
-
-func validateStore(cluster opt.Cluster, id uint64) error {
```

Comment:
Can't use `ClusterInfo` to replace `opt.Cluster`?

Reply:
We can but I think it's unnecessary.